.talismanrc

```diff
@@ -1,212 +1,216 @@
 fileignoreconfig:
-    - filename: .github/workflows/staging-ci.yaml
```

Comment:
I don't think I have ever seen so many talisman changes, what happened?

Reply:
I'm honestly not sure. It did that when I went to commit my changes.

Reply:
I manually reverted talisman and when it ran again it didn't prompt for changes this time. It looks like previously it had messed up the formatting / indentation and added some test files to `.talismanrc`.